src/script/app/eventemitter.ts

```diff
@@ -0,0 +1,8 @@
+import mitt from 'mitt';
```

Comment:
This is the file that is responsible for initializing the event emitter and configure all the possible events and payload types.